### ISSUE TITLE
failed to run example golang/web_server on ubuntu 18.04

### ISSUE THREAD

**Issue by kunmel** [occlum/occlum#610]
After installed occlum and occlum-go on ubuntu18.04, I was trying to build the 'golang/web_server' but it failed.
1. downgraded the kernel to 4.15.0 and installed enable_rdfsbase as suggested.
2. hello_world demo runs well, but all golang demos have the same problem.

Here is the log of 'web_server':
```
/home/guotiezheng/go/src/github.com/occlum/demos/golang/web_server/occlum_instance initialized as an Occlum instance
Enclave sign-tool: /opt/occlum/sgxsdk-tools/bin/x64/sgx_sign
Enclave sign-key: /opt/occlum/etc/template/Enclave.pem
SGX mode: HW
Building new image...
[+] Home dir is /home/guotiezheng
[+] Open token file success! 
[+] Token file valid!
[+] Init Enclave Successful 22621592748034!
Generate the SEFS image successfully
EXPORT => OCCLUM_CONF_FILE_MAC = 86-a1-88-b5-4f-51-6c-ff-da-d1-6a-12-3f-0d-95-11
Building the initfs...
[+] Home dir is /home/guotiezheng
[+] Open token file success! 
[+] Token file valid!
[+] Init Enclave Successful 22690312224770!
Generate the SEFS image successfully
Building libOS...
Signing the enclave...
<EnclaveConfiguration>
    <ProdID>0</ProdID>
    <ISVSVN>0</ISVSVN>
    <StackMaxSize>1048576</StackMaxSize>
    <StackMinSize>1048576</StackMinSize>
    <HeapMaxSize>33554432</HeapMaxSize>
    <HeapMinSize>33554432</HeapMinSize>
    <TCSNum>32</TCSNum>
    <TCSPolicy>1</TCSPolicy>
    <DisableDebug>0</DisableDebug>
    <MiscSelect>0</MiscSelect>
    <MiscMask>0xFFFFFFFF</MiscMask>
    <ReservedMemMaxSize>1048576000</ReservedMemMaxSize>
    <ReservedMemMinSize>1048576000</ReservedMemMinSize>
    <ReservedMemInitSize>1048576000</ReservedMemInitSize>
    <ReservedMemExecutable>1</ReservedMemExecutable>
</EnclaveConfiguration>
tcs_num 32, tcs_max_num 32, tcs_min_pool 1
The required memory is 1118646272B.
The required memory is 0x42ad3000, 1092428 KB.
Succeed.
Built the Occlum image and enclave successfully
occlum run /bin/web_server
/opt/occlum/build/bin/occlum: line  302:  5335 Illegal instruction     (core dumped) RUST_BACKTRACE=1 "$instance_dir/build/bin/occlum-run" "$@"
```

**Comment by kunmel** [comment]
the go version is 1.13.7

**Comment by guzongmin** [comment]
@kunmel What is the Occlum version you are using?

**Comment by kunmel** [comment]
Thank you for your reply. The Occlum version is 0.21.0. 
And after I run 'OCCLUM_LOG_LEVEL=trace occlum run /bin/web_server' the log is:
[log.txt](https://github.com/occlum/occlum/files/7038434/log.txt)

**Comment by kunmel** [comment]
@guzongmin It works in docker mod with the latest image, so maybe it's because the version I installed is too old?

**Comment by guzongmin** [comment]
@kunmel Can you please try 0.23.7? We have solved a lot of issue in the old version.

**Comment by guzongmin** [comment]
> @guzongmin It works in docker mod with the latest image, so maybe it's because the version I installed is too old?

You could get the 0.23.7 docker image on dockerhub.

**Comment by kunmel** [comment]
Ok, thank you for your help, I will try the 0.23.7.

**Comment by kunmel** [comment]
@guzongmin  Hi, I tried it on 0.23.7 in docker mod and the demo runs well. 
But I got the following error when I setup the docker container, maybe it's because my machine  supports only SGX1 but not FLC feature? 
How can I solve this problem？
```
root@133942f4ec7e:~# aesm_service[13]: [ADMIN]White List update requested
aesm_service[13]: [load_qe ../qe_logic.cpp:641] Error, call sgx_create_enclave QE fail [load_qe], SGXError:4004.

aesm_service[13]: Failed to load QE3: 0x4004
aesm_service[13]: The server sock is 0x5626149d2a60
aesm_service[13]: [ADMIN]White list update request successful for Version: 103

```

**Comment by guzongmin** [comment]
> @guzongmin Hi, I tried it on 0.23.7 in docker mod and the demo runs well.
> But I got the following error when I setup the docker container, maybe it's because my machine supports only SGX1 but not FLC feature?
> How can I solve this problem？
> 
> ```
> root@133942f4ec7e:~# aesm_service[13]: [ADMIN]White List update requested
> aesm_service[13]: [load_qe ../qe_logic.cpp:641] Error, call sgx_create_enclave QE fail [load_qe], SGXError:4004.
> 
> aesm_service[13]: Failed to load QE3: 0x4004
> aesm_service[13]: The server sock is 0x5626149d2a60
> aesm_service[13]: [ADMIN]White list update request successful for Version: 103
> ```

Are you able to run the demo? I think the QE3 issue should not impact you.

**Comment by kunmel** [comment]
> > @guzongmin Hi, I tried it on 0.23.7 in docker mod and the demo runs well.
> > But I got the following error when I setup the docker container, maybe it's because my machine supports only SGX1 but not FLC feature?
> > How can I solve this problem？
> > ```
> > root@133942f4ec7e:~# aesm_service[13]: [ADMIN]White List update requested
> > aesm_service[13]: [load_qe ../qe_logic.cpp:641] Error, call sgx_create_enclave QE fail [load_qe], SGXError:4004.
> > 
> > aesm_service[13]: Failed to load QE3: 0x4004
> > aesm_service[13]: The server sock is 0x5626149d2a60
> > aesm_service[13]: [ADMIN]White list update request successful for Version: 103
> > ```
> 
> Are you able to run the demo? I think the QE3 issue should not impact you.

Yes, I can run the golang demos. So should I just ignore this QE3 error? Because I don't know if it will have an impact.
